fetch.bs

```diff
@@ -5889,11 +5889,20 @@ run these steps:
    the <a data-lt="structured field token">token</a> <code>prefetch</code>) in
    <var>httpRequest</var>'s <a for=request>header list</a>.
 
-   <li><p>If <var>httpRequest</var>'s <a for=request>header list</a>
-   <a for="header list">does not contain</a> `<code>User-Agent</code>`, then user agents should
-   <a for="header list">append</a> (`<code>User-Agent</code>`,
-   <a>default `<code>User-Agent</code>` value</a>) to <var>httpRequest</var>'s
-   <a for=request>header list</a>.
+   <li>
+    <p>If <var>httpRequest</var>'s <a for=request>header list</a> <a for="header list">does not
+    contain</a> `<code>User-Agent</code>`, then:
+
+    <ol>
+     <li><p>Let <var>userAgent</var> be the result of running the <a>WebDriver BiDi emulated
+     User-Agent</a> with <var>httpRequest</var>'s <a for=request>client</a>.
```

Comment:
No newlines inside `<a>`.

Reply:
There's also a problem in that WebDriver hands us a string but we need a byte sequence.

Reply:
I assume instead it should be:
> [UTF-8 encoding](https://encoding.spec.whatwg.org/#utf-8-encode) of <var>userAgent</var>?

Reply:
Oh, apparently it is "[isomorphic encoded](https://infra.spec.whatwg.org/#isomorphic-encode)"